### ISSUE TITLE
getting error

### ISSUE THREAD

**Issue by humph843** [MicrosoftDocs/cpp-docs#4662]
So when running the code as instructed, i.e. pasting the code example into VS and building, I get a LNK2001 error code for unresolved external symbol _main.  Referencing the MSVCRT.lib(exe)main.obj file.  Running this in new install of VS 2022 with C++ empty project template.

not sure what I missed, but there doesn't seem to be help in the current page for resolution.  Probably a setting or a missing include file??

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: eda99bc7-483b-e53c-1035-886f4d0955f4
* Version Independent ID: a6fdba4d-53e4-9f1a-2352-bc68ef491069
* Content: [Walkthrough: Create a traditional Windows Desktop application (C++)](https://learn.microsoft.com/en-us/cpp/windows/walkthrough-creating-windows-desktop-applications-cpp?view=msvc-170#build-the-code)
* Content Source: [docs/windows/walkthrough-creating-windows-desktop-applications-cpp.md](https://github.com/MicrosoftDocs/cpp-docs/blob/main/docs/windows/walkthrough-creating-windows-desktop-applications-cpp.md)
* Product: **visual-cpp**
* Technology: **cpp-windows**
* GitHub Login: @TylerMSFT
* Microsoft Alias: **twhitney**

**Comment by TylerMSFT** [comment]
Hi @humph843 - I took a look and this article needs to be updated. I'll add overhauling this to work with the latest toolset to my list. Thank you for pointing it out. Given my current backlog, it may be a bit before I can get to this.

**Comment by humph843** [comment]
Thanks for the quick reply Tyler.  Didn't expect this.  Just a follow up. After some scouring, I found that my attempt somehow had the program properties set up for a console app, rather than a Windows app.  Once corrected, this fixed the issue.  Now I need to figure our why it set up as a console.  Could swear I specified a Windows empty project when creating the new project.  Still learning and probably my mistake.    Thanks again!

**Comment by TylerMSFT** [comment]
The fact that you ran into this snag means I need to give this article a good going over to make sure others don't hit the same thing. Really appreciate your follow up.

**Comment by TylerMSFT** [comment]
I just ran through the steps and it worked w/o error. I strengthened some comments about waiting until the end to copy the finished project to run it. I also added a statement to make sure to use the Windows Desktop Wizard and choose the empty project type, not any other type.  I also updated the alt-text for the graphics so that the vision-impaired will have an easier time of it. These updates will be available later this week.
